pkg/scheduler/core/generic_scheduler.go

```diff
@@ -201,6 +201,12 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 		return result, err
 	}
 
+	// Run "prefilter" plugins.
```

Comment:
Please move this block to line 192, right after the `podPassesBasicChecks` block. In fact, `podPassesBasicChecks` should be eventually converted to a default prefilter plugin.

Reply:
Done.

Reply:
@bsalamat I was wondering why the pre-filter plugins where placed between the PVC check (that's what `podPassesBasicCheck` is), and the there-is-at-least-one-node check and snapshot update. I found this comment where you specifically requested to move it there and suggest that the first (PVC check) should be converted to a pre-filter. Would the latter two also fall into this category?